apm-protocol/apm-network/pom.xml

```diff
@@ -52,11 +52,23 @@
             <groupId>io.grpc</groupId>
             <artifactId>grpc-protobuf</artifactId>
             <version>${grpc.version}</version>
+            <exclusions>
+                <exclusion>
+                    <artifactId>netty-codec-http2</artifactId>
+                    <groupId>io.netty</groupId>
+                </exclusion>
+            </exclusions>
```

Comment:
I can't see `netty-codec-http2` in the dependency tree of this. Why do you do this?